tests/public-api/tutorials/sleeper.py

```diff
@@ -0,0 +1,92 @@
+## pip install git+https://github.com/ITISFoundation/osparc-simcore-python-client.git@is1865/apirunners
+##
+## export MY_API_KEY=***
+## export MY_API_SECRET=***
+## python sleeper.py
+##
+
+import os
+import time
+from pathlib import Path
+
+import osparc
+from osparc import UsersApi
+from osparc.api import FilesApi, SolversApi
+from osparc.models import File, Job, JobInputs, JobOutputs, JobStatus, Solver
+
+Path("file_with_number.txt").write_text("3")
+
+
+cfg = osparc.Configuration(
+    host=os.environ.get("OSPARC_API_URL", "http://127.0.0.1:8006"),
+    username=os.environ["MY_API_KEY"],
+    password=os.environ["MY_API_SECRET"],
+)
+
+with osparc.ApiClient(cfg) as api_client:
+
+    profile = UsersApi(api_client).get_my_profile()
+    print(profile)
+    #
+    #  {'first_name': 'foo',
+    #  'gravatar_id': 'aa33fssec77ea434c2ea4fb92d0fd379e',
+    #  'groups': {'all': {'description': 'all users',
+    #                     'gid': '1',
+    #                     'label': 'Everyone'},
+    #             'me': {'description': 'primary group',
+    #                    'gid': '2',
+    #                    'label': 'foo'},
+    #             'organizations': []},
+    #  'last_name': '',
+    #  'login': 'foo@itis.swiss',
+    #  'role': 'USER'}
+    #
+
+    files_api = FilesApi(api_client)
+    input_file: File = files_api.upload_file(file="file_with_number.txt")
+
+    solvers_api = SolversApi(api_client)
+    solver: Solver = solvers_api.get_solver_release(
+        "simcore/services/comp/itis/sleeper", "2.0.2"
```

Comment:
one question here and this might be an issue or not. sleeper:2.0.2 is accessible by all right? that is why that test works. sorry was confused since I think the latest sleeper has access rights that should prevent it from running. since you use the catalog to get the solvers I guess a user cannot run a service that he/she has no access to?

Reply:
yes, it is actually wrong. I have just created a session fixture that sets up the registry with sleeper:2.1.1 and takes ownership of the image by overriding labels pushing to the registry